contrib/go-chi/chi/option.go

```diff
@@ -71,3 +74,15 @@ func WithAnalyticsRate(rate float64) Option {
 		}
 	}
 }
+
+// WithIsErrorCheck sets the function which is used to decide whther the
+// status should be marked as an error
+func WithIsErrorCheck(isError func(statusCode int) bool) Option {
```

Comment:
I'm more inclined towards something like `WithStatusCheck` or `WithErrorCheck`. `WithIsErrorCheck` is a little cumbersome.

What do you think?

Reply:
`WithStatusCheck` Sounds better to me too.